assets/css/main.css

```diff
@@ -3,13 +3,152 @@
   -moz-box-sizing: border-box;
   box-sizing: border-box; }
 
-.container {
+h1, h2, h3, h4, h5, h6,
+ul, li,
+p {
+  padding: 0;
+  margin: 0; }
+
+h1, h2, h3, h4, h5, h6 {
+  font-weight: normal; }
+
+body {
+  background-color: #450081;
+  color: #ffffff;
+  font-family: Lato, Arial, sans-serif; }
+  body a {
```

Comment:
Please add single quotes around `Lato` and make the next default front `Helvetica`.